lxd/storage_lvm.go

```diff
@@ -88,6 +88,24 @@ func storageLVExists(lvName string) (bool, error) {
 	return true, nil
 }
 
+func lvmGetLVSize(lvPath string) (string, error) {
+	msg, err := shared.TryRunCommand("lvs", "--noheadings", "-o", "size", "--nosuffix", "--units", "b", lvPath)
```

Comment:
Retrieving the value in bytes here, since to prevent issues like `lvs` resporting something like `0.5GB`.